### ISSUE TITLE
@sentry/cli binary doesn't download (leads to iOS build failure)

### ISSUE THREAD

**Issue by capella55** [getsentry/sentry-react-native#2688]
**Environment**
os: macOS 12.6
platform: iOS
yarn: 1.22.19
react: 17.0.2
react-native: 0.68.2

I created a new React-Native project by running `npx react-native init AwesomeProject --version 0.68.0`.
After that, I ran the following commands to install `Sentry`.

```
yarn add @sentry/react-native
npx @sentry/wizard -i reactNative -p ios android
cd ios
pod install
```

**I have the following issue:**
```
Android works well but I see the error while building the iOS app.
Command PhaseScriptExecution failed with a nonzero exit code

Error: spawn /AwesomeProject/node_modules/@sentry/cli/sentry-cli ENOENT
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn /AwesomeProject/node_modules/@sentry/cli/sentry-cli',
  path: '/Volumes/DATA/Project/AwesomeProject/node_modules/@sentry/cli/sentry-cli',
  spawnargs: [ 'upload-dsym' ]
}
Command PhaseScriptExecution failed with a nonzero exit code

```

**Comment by krystofwoldrich** [comment]
Hi,
it looks like the path to the CLI executable is wrong.

It should be `../node_modules/@sentry/cli/bin/sentry-cli`.

Can you try to replace it in the `Upload Debug Symbols to Sentry` Build Phase? Also could you send how the phase looks like at the moment? Could you also send version of the wizard using command `npx @sentry/wizard  --version`.

**Comment by jeremyfalldev** [comment]
@krystofwoldrich Thank you for checking this!
I confirmed @sentry/cli@1.74.4 was installed already. However, I ran the above and tried again but saw the same issue.

Here is a screenshot.

![screenshot](https://user-images.githubusercontent.com/86626192/207144972-1548374a-3af3-4f89-a75b-fb6b5459c139.png)

I see @sentry/cli but not @sentry/cli/sentry-cli. Is this a problem?

**Comment by capella55** [comment]
@krystofwoldrich @jeremyfalldev Same issue on my side. :(

**Comment by krystofwoldrich** [comment]
@capella55 @jeremyfalldev For some reason the CLI install script didn't download the CLI binary or didn't get executed.

Try to execute it manually.

```bash
cd node_modules/@sentry/cli
node ./scripts/install.js
```

**Comment by capella55** [comment]
@krystofwoldrich Yes, it seems like the CLI install script does not work properly. Please check this screenshot.
![Screen Shot 2022-12-13 at 11 24 40 AM](https://user-images.githubusercontent.com/15838658/207426315-7f3a18dd-bb5a-4ab0-be2f-1bace196ff9a.png)
It has been never completed 100% and I don't see `sentry-cli`.

**Comment by krystofwoldrich** [comment]
Are you able to download the file from the URL manually?

https://downloads.sentry-cdn.com/sentry-cli/1.74.4/sentry-cli-Darwin-universal

@capella55 So the `install.js` never finishes?

@kamilogorek Do you know what could cause this?

**Comment by kamilogorek** [comment]
I tried to replicate the issue locally but with no luck. I see that your output is not TTY, as then it'd not print multiple progressbar lines (it shouldn't anyway as bar should be rendered in TTY environments only anyway ¯\_(ツ)_/¯).

I wonder if it may have something to do with it. Can you try to run it with `SENTRYCLI_NO_PROGRESS_BAR=1` instead?

**Comment by robgraeber** [comment]
Same issue using a yarn monorepo, running `install.js` manually fixes it.

**Comment by robgraeber** [comment]
Hmm I tried reproducing it in another repo but can't reliably. I also deleted my `.yarn/cache` folder in my original project and it seemed to reinstall successfully. I'm using yarn workspaces, yarn 3.2.1, node 16, and a mac. 

Here's the repo of what I'm doing, it reproduced one time but doesn't happen when I try it again. https://github.com/robgraeber/sentry-test

**Comment by krystofwoldrich** [comment]
@robgraeber Thanks for the repo I'll try it and let you know if I managed to reproduce it.

**Comment by krystofwoldrich** [comment]
@robgraeber I'm sorry for the wait, sadly I still can't reproduce the issue. I've tried the repo with the same version of node and yarn, but no luck.

**Comment by krystofwoldrich** [comment]
I will close this for now, as we are not able to reproduce the issue, if there would be some new context we could try feel free to inform us, and we can reopen the issue and continue with it.

**Comment by grossadamm** [comment]
In case some internet searcher comes along in the future...

I'm also using `bun` and I struggled with this for a while. I had the same issue where `install.js` was not being triggered and the sentry-cli rust executable was not present. This results in a file not found error in the spawn command which is very cryptic and confusing to debug until you dig into the repo a bit.

Not sure what precisely solved it for me, but it works now. Here's everything I did.

I moved `@sentry/cli` into dependencies from devDependencies and ensured that `trustedDependencies` listed it.

```
"trustedDependencies": [
    "@sentry/cli"
  ]
```

Then these commands

```
bun upgrade
rm -rf ~/.bun/install/cache
 rm -rf node_modules bun.lockb
```
